wire-library/wire-gradle-plugin/src/test/projects/protopath-ordering/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+  id 'org.jetbrains.kotlin.jvm'
+  id 'com.squareup.wire'
+}
+
+wire {
+  protoPath {
+    srcDir("src/main/proto")
+    include("squareup/dinosaurs/dig.proto")
+    include("squareup/geology/period.proto")
+    include("squareup/dinosaurs/dinosaur.proto")
+  }
+
+  kotlin {
+  }
+}
+
+afterEvaluate {
+  tasks.withType(com.squareup.wire.gradle.WireTask).each { t ->
+    def wireInputs = t.protoInput$wire_gradle_plugin.get().collect({ v -> v.path })
+    println "${t.identityPath} protoInputs = ${wireInputs}"
+  }
+}
```

Comment:
```suggestion
```

Reply:
This part will output in the buid log what will be checked then by the test assertion. Removing it will make the test fail, do you suggest maybe to remove the test altogether ?

Reply:
oh I see. Hmmm let me think a bit.

Reply:
I think we could remove the tests from this PR since they are both testing implementation details.
If we ever find a way to test this properly, we'll use that but not essential for now.

Reply:
Thanks, sounds good ! I've just removed the test